.github/workflows/docker.yaml

```diff
@@ -0,0 +1,40 @@
+# GitHub actions workflow which builds and publishes the docker images.
+
+name: Build docker images
+
+on:
+  push:
+    tags: ["v*"]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Log in to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
The secret is called `DOCKER_HUB_USERNAME`.

I can't see a `DOCKERHUB_TOKEN` secret in this repo's settings. You might have to request that it's shared with this repo.

Reply:
> The secret is called DOCKER_HUB_USERNAME.

Looking at synapse: oddly we seem to have an org-level secret with the extra underscore, but repo-level secrets without the underscore.

Reply:
Fun fun fun. Sharing request has been made here, waiting on ops to action it.